### PR TITLE
Added brands of Kuups Desging International and mathcNames and matchTags from ALE-HOP

### PR DESCRIPTION
**Pull request by paunofu**
This pull request adds the three brands that Kuups Design International has in Spain and the matchTags and machName from ALE-HOP

### CODE REVIEW

File: data/brands/shop/gift.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
     {
       "displayName": "ALE-HOP",
       "id": "alehop-be38fa",
-      "locationSet": {
-        "include": ["es", "hr", "mx", "pt"]
-      },
+      "locationSet": {"include": ["es", "hr", "mx", "pt"]},
+      "matchNames": ["Ale-Hop","ale-hop","Alehop"],
+      "matchTags": ["shop/variety_store","shop/clothes","shop/accessories"],
```
Comment (LaoshuBaby) [inline data/brands/shop/gift.json:28]: It is `shop=fashion_accessories`, not `shop=accessories`

File: data/brands/shop/convenience.json [hunk 2]
```diff
@@ -1674,6 +1674,20 @@
...
+    {
+      "displayName": "Kuups",
+      "locationSet":{"include":["es"]},
+      "matchNames":["Kups"],
+      "matchTags":["shop/supermarket"]
+      "tags":{
+        "brand":"Kuups",
+        "name": "Kuups",
+        "operator": "Kuups Design International SL",
+        "operator:wikidata": "Q113464694",
```
Comment (LaoshuBaby) [inline data/brands/shop/convenience.json:1686]: This entry is in brand tree, so you must add a brand:wikidata for Kuups or it won't be suggested.

Comment (LaoshuBaby) [inline data/brands/shop/convenience.json:1686]: This entry is in brand tree, so you must add a brand:wikidata for Kuups or it won't be suggested.

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/shop/supermarket.json [hunk 3]
```diff
@@ -1855,6 +1855,17 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Economy Cash",
+      "locationSet": {"include":["es"]},
+      "tags":{
+        "brand": "Economy Cash",
+        "operator": "Kuups Design International SL",
```
Comment (LaoshuBaby) [inline data/brands/shop/supermarket.json:1863]: The same problem as https://github.com/osmlab/name-suggestion-index/pull/7274/files#r1008737352

(You don't need to remove operator/operator:wikidata)

Below "Vidal Supermercados" also missing a brand:wikidata

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/shop/convenience.json (hunk 2 above)
Comment (paunofu) [reply data/brands/shop/convenience.json:1686]: Well, they don’t have a wikidata page, so I guess I have to create it. Thank you for your reply.

**Pull request merged**